.gitignore

```diff
@@ -35,3 +35,4 @@ gwt-unitCache/
 class_list.txt
 *.jpage
 .factorypath
+dependency-reduced-pom.xml
```

Comment:
Hi @wmedvede, is this line necessary or did it slip in? :)

Reply:
@tiagobento I think it's necessary since this file is generated by the shade plugin